cdap-docs/examples-manual/build.sh

```diff
@@ -62,7 +62,13 @@ function download_file() {
 
   echo "Downloading using curl $file_name from $source_dir"
   curl $source_dir/$file_name --output $target --silent
-  local new_md5_hash=`md5 -q $target`
+
+  if [ "$OSTYPE" == "darwin"* ]; then
+    local new_md5_hash=`md5 -q $target`
+  else
+    local new_md5_hash=`md5sum $target | awk '{print $1}'`
+  fi
+
   if [ "x$md5_hash" != "x$new_md5_hash" ]; then
     echo -e "$WARNING MD5 Hash for $file_name has changed! Compare files and update hash!"  
     echo "Old md5_hash: $md5_hash New md5_hash: $new_md5_hash"
```

Comment:
What's really nice is to have this line as:
```
echo -e "Old MD5 Hash: ${md5_hash} New MD5 Hash: ${RED}${BOLD}${new_md5_hash}${NC}"
```
then the new MD5 hash is in red and easy to find and copy.